rd6006/rd6006.py

```diff
@@ -131,10 +152,59 @@ def enable(self):
     @enable.setter
     def enable(self, value):
         self._write_register(18, int(value))
+    
+    @property
+    def ocpovp(self):
+        m = self._read_register(16)
```

Comment:
You should directly return the value here instead of setting a variable (more consistency)

Reply:
yes, agree